src/main/java/com/google/devtools/build/lib/starlarkbuildapi/objc/AppleBootstrap.java

```diff
@@ -29,6 +29,7 @@ public class AppleBootstrap implements Bootstrap {
       ImmutableSet.of(
           PackageIdentifier.createUnchecked("_builtins", ""),
           PackageIdentifier.createUnchecked("bazel_tools", ""),
+          PackageIdentifier.createUnchecked("build_bazel_rules_apple", ""), // alias for rules_apple
```

Comment:
I can see this list if much shorter than the one at HEAD, @keith @brentleyjones can you confirm if other repos are also needed here?

Reply:
OK, probably dd02b9e52206499ede851a2113fd82a533ee4f67 should have been backported also,

Reply:
yea i think so